<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatItem</class>
 <widget class="QWidget" name="ChatItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_user_avatar">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_user_name">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>16</y>
     <width>110</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_content">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>39</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="RedPoint" name="lbl_redpoint">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>5</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_last_time">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>13</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RedPoint</class>
   <extends>QLabel</extends>
   <header>RedPoint.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
